docs/2.7.0/docs/daml-finance/tutorials/settlement/internal.rst

```diff
@@ -0,0 +1,153 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Internal Settlement
+###################
+
+This tutorial builds upon the concepts introduced in the
+:doc:`Settlement <../getting-started/settlement>` getting-started tutorial. Compared to our previous
+:doc:`Enhanced Transfers <transfer>` tutorial, which demonstrated the "direct" transfer a holding
+between sender to receiver (at a single custodian), it delves further into the settlement mechanism
+with a batch and related instructions. This intricate process enables the adjustment of record books
+across multiple entities and instrument holdings simultaneously.
+
+In this tutorial, we will limit the complexity by focusing on a single custodian and the transfer of
+a singular instrument. The next tutorial will explore the broader scenario involving multiple
+custodians. Eager learners are encouraged to extend this tutorial and the following one by
+incorporating more than one instrument as an exercise.
+
+Understanding Internal Settlement with Examples
+***********************************************
+
+To start, let us briefly revist the concept of a
+:ref:`Batch <type-daml-finance-interface-settlement-batch-batch-97497>`, which consists of one or
+more :ref:`Instructions <type-daml-finance-interface-settlement-instruction-instruction-30569>`.
+Each instruction signifies a
+:ref:`RoutedStep <type-daml-finance-interface-settlement-types-routedstep-10086>`, delineating the
+quantity of an instrument to be transferred from a sender to a receiver at a specific custodian. For
```

Comment:
fyi, you could also link to the `Settlement` page in the concepts section

Reply:
added a link